js/screens/NotificationsScreen.js

```diff
@@ -140,12 +137,9 @@ class NotificationsScreen extends React.Component {
   }
 
   _openNotificationForSite(notification, site) {
-    site
-      .readNotification(notification)
-      .catch(e => {
-        console.log('failed to mark notification as read ' + e);
-      })
-      .done();
+    site.readNotification(notification).catch(e => {
+      console.log('failed to mark notification as read ' + e);
```

Comment:
Looks like this was in the previous version too. Should we have console.log here?